.gitignore

```diff
@@ -1,3 +1,4 @@
 .gradle
 build/
-!gradle/wrapper/gradle-wrapper.jar
\ No newline at end of file
+!gradle/wrapper/gradle-wrapper.jar
+.idea/
```

Comment:
please add to your own gitignore ;)https://bgalek.gitbook.io/askbartosz/git#set-global-gitignore